### ISSUE TITLE
Two Devices

### ISSUE THREAD

**Issue by rogiervdberg** [AlexanderBabel/homebridge-broadlink-rm#57]
When I try to add a second device it only shows me one in the home app. 
It only adds the first accessory. Is this a known issue? 

This is my config file: 

            "platform": "BroadlinkRM",
            "hosts": [
                {
                    "address": "192.168.1.241",
                    "mac": "c8:f7:42:17:46:f8"
                }
            ],
            "name": "Broadlink RM",
            "hideScanFrequencyButton": false,
            "hideLearnButton": false,
            "hideWelcomeMessage": false,
            "accessories": [
                {
                    "name": "Radio",
                    "type": "tv",
                    "pingIPAddressStateOnly": true,
                    "pingFrequency": 3,
                    "pingGrace": 24,
                    "data": {
                        "on": "260050000001279412381139113812131214111411141238111411141238113812381139113812131237121412381114121312381114121312141138121411381238111411391138120004fb0001264c11000d050000000000000000",
                        "off": "26004800000127951138123811391114111412131214113812131214113812381139113811391114111412131214113812141114113812381139113812381114113911381213121411000d05",
                        "remote": {
                            "select": "AA",
                            "arrowUp": "AA",
                            "arrowDown": "AA",
                            "arrowLeft": "AA",
                            "arrowRight": "AA",
                            "back": "AA",
                            "exit": "AA",
                            "playPause": "AA",
                            "info": "AA"
                        },
                        "volume": {
                            "up": "260050000001269512381138123811141214111411141139111411141139113812381139113812131214111411141139111411141213123811391138123811141139113812381114110004fc0001264c12000d050000000000000000",
                            "down": "260050000001269512381138123811141114121411141138121411141139113812381138123811141213121411381238111411141214113812381138121411141138123811391114110004fc0001264c11000d050000000000000000"
                        },
                        "inputs": [
                            {
                                "name": "TV",
                                "type": "hdmi",
                                "data": "260050000001289314351436143613121411141213121336141213121336143614361336143613121436133614121336143613121411143613121411143613121411143614361312130004fa0001284914000d050000000000000000"
                            },
                            {
                                "name": "Airplay",
                                "type": "hdmi",
                                "data": "260050000001279511381238113911141114111412141138121114141138123811391138113911141114111412141114111411141238113911381238113812381139113812141114110004fb0001274b12000d050000000000000000"
                            }
                        ]
                    }
                },
                {
                    "name": "TV",
                    "type": "tv",
                    "pingIPAddressStateOnly": true,
                    "pingFrequency": 3,
                    "pingGrace": 24,
                    "data": {
                        "on": "260050000001279412381139113812131214111411141238111411141238113812381139113812131237121412381114121312381114121312141138121411381238111411391138120004fb0001264c11000d050000000000000000",
                        "off": "26004800000127951138123811391114111412131214113812131214113812381139113811391114111412131214113812141114113812381139113812381114113911381213121411000d05",
                        "remote": {
                            "select": "AA",
                            "arrowUp": "AA",
                            "arrowDown": "AA",
                            "arrowLeft": "AA",
                            "arrowRight": "AA",
                            "back": "AA",
                            "exit": "AA",
                            "playPause": "AA",
                            "info": "AA"
                        },
                        "volume": {
                            "up": "260050000001269512381138123811141214111411141139111411141139113812381139113812131214111411141139111411141213123811391138123811141139113812381114110004fc0001264c12000d050000000000000000",
                            "down": "260050000001269512381138123811141114121411141138121411141139113812381138123811141213121411381238111411141214113812381138121411141138123811391114110004fc0001264c11000d050000000000000000"
                        },
                        "inputs": [
                            {
                                "name": "TV",
                                "type": "hdmi",
                                "data": "260050000001289314351436143613121411141213121336141213121336143614361336143613121436133614121336143613121411143613121411143613121411143614361312130004fa0001284914000d050000000000000000"
                            },
                            {
                                "name": "Airplay",
                                "type": "hdmi",
                                "data": "260050000001279511381238113911141114111412141138121114141138123811391138113911141114111412141114111411141238113911381238113812381139113812141114110004fb0001274b12000d050000000000000000"
                            }
                        ]
                    }
                }
            ]
        }

Thanks!

**Comment by AlexanderBabel** [comment]
If you are adding more than one TV to your configuration only the first one is directly exposed. Each following TV must be added separately to the Home due to a restriction of one TV per bridge/accessory.Just take a look in your log of the homebridge instance and follow the instructions there.

TL;DR: Basically you‘ll find the other TV in the add accessory screen in the Home app. The PIN is the same as for your homebridge instance.

**Comment by rogiervdberg** [comment]
I got it working yesterday! Thank you so much for this plugin.

Thanks!
Rogier van den Berg
Rogier-berg@hotmail.com

On 30 Nov 2019, at 14:40, Alexander Babel <notifications@github.com> wrote:

﻿

If you are adding more than one TV to your configuration only the first one is directly exposed. Each following TV must be added separately to the Home due to a restriction of one TV per bridge/accessory.Just take a look in your log of the homebridge instance and follow the instructions there.

TL;DR: Basically you‘ll find the other TV in the add accessory screen in the Home app. The PIN is the same as for your homebridge instance.
